### PR TITLE
zib-ComfortScale

### PR DESCRIPTION
**Pull request by furoreSebastiaan**

### CODE REVIEW

File: Profiles - ZIB 2017/zib-ComfortScale.xml [hunk 1]
```diff
@@ -0,0 +1,380 @@
...
+      <path value="Observation.code.coding.code" />
+      <min value="1" />
+      <fixedCode value="108301000146109" />
+    </element>
+    <element id="Observation.code.coding:comfortScoreCoding.display">
+      <extension url="http://hl7.org/fhir/StructureDefinition/elementdefinition-translatable">
+        <valueBoolean value="true" />
+      </extension>
+      <path value="Observation.code.coding.display" />
+      <defaultValueString value="MISSING" />
```
Comment (pieter-edelman-nictiz) [inline Profiles - ZIB 2017/zib-ComfortScale.xml]: Do you have a default value?

File: Profiles - ZIB 2017/zib-ComfortScale.xml [hunk 2]
```diff
@@ -0,0 +1,380 @@
...
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="code" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+      <min value="1" />
+    </element>
+    <element id="Observation.code.coding:comfortScoreCoding">
```
Comment (pieter-edelman-nictiz) [inline Profiles - ZIB 2017/zib-ComfortScale.xml]: Slice should be called 'comfortScaleCoding'. 'ComfortScore' is the Dutch name.

File: Examples/zib-ComfortScale - example.xml [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+            <code value="108301000146109" />
+            <system value="http://snomed.info/sct" />
+            <display value="ComfortScale" />
+        </coding>
+    </code>
+    <subject>
+        <reference value="Patient/example"/>
+        <display value="Example Patient"/>
+    </subject>
+    <effectiveDateTime value="2019-09-03T11:01:07.9477128+00:00" />
```
Comment (ArdonToonstra) [inline Examples/zib-ComfortScale - example.xml]: Don't we have total score?

File: Profiles - ZIB 2017/zib-ComfortScale.xml [hunk 4]
```diff
@@ -0,0 +1,413 @@
...
+  <publisher value="Nictiz" />
+  <contact>
+    <name value="Nictiz" />
+    <telecom>
+      <system value="email" />
+      <value value="info@nictiz.nl" />
+      <use value="work" />
+    </telecom>
+  </contact>
+  <description value="The Comfort Behaviour scale /Comfortneo scale is an assessment tool, that is used to assess the pain behaviour of neonates (new born).&#xD;&#xA;The Comfort Behaviour scale has 7 observation points that that need to be addressed. For each on observation is value is assigned from a 5-point scale, the final score indicates whether the observed child suffers pain and 'how much'. The score is combined with an NRS score that is completed by the nurses on basis of clinical experience." />
```
Comment (ArdonToonstra) [inline Profiles - ZIB 2017/zib-ComfortScale.xml]: Please follow profiling guidelines

File: Profiles - ZIB 2017/zib-ComfortScale.xml [hunk 5]
```diff
@@ -0,0 +1,413 @@
...
+      <definition value="Observation of:  Alertness" />
+      <alias value="Alertheid" />
+      <type>
+        <code value="CodeableConcept" />
+      </type>
+      <binding>
+        <strength value="extensible" />
+        <description value="AlertheidCodelijst" />
+        <valueSetReference>
+          <reference value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.40.2.12.12.1--20171231000000" />
```
Comment (ArdonToonstra) [inline Profiles - ZIB 2017/zib-ComfortScale.xml]: Please add valuesetReferene displays for all

File: Profiles - ZIB 2017/zib-ComfortScale.xml [hunk 6]
```diff
@@ -0,0 +1,413 @@
...
+    <element id="Observation.component:BodyMovement">
+      <path value="Observation.component" />
+      <sliceName value="BodyMovement" />
+      <max value="1" />
+    </element>
+    <element id="Observation.component:BodyMovement.code">
+      <path value="Observation.component.code" />
+      <patternCodeableConcept>
+        <coding>
+          <system value="urn:oid:2.16.840.1.113883.2.4.3.11.60.40.4.0" />
```
Comment (ArdonToonstra) [inline Profiles - ZIB 2017/zib-ComfortScale.xml]: MSSING CODE!

File: Profiles - ZIB 2017/zib-ComfortScale.xml [hunk 7]
```diff
@@ -0,0 +1,413 @@
...
+    <element id="Observation.component:FacialTone">
+      <path value="Observation.component" />
+      <sliceName value="FacialTone" />
+      <max value="1" />
+    </element>
+    <element id="Observation.component:FacialTone.code">
+      <path value="Observation.component.code" />
+      <patternCodeableConcept>
+        <coding>
+          <system value="urn:oid:2.16.840.1.113883.2.4.3.11.60.40.4.0" />
```
Comment (ArdonToonstra) [inline Profiles - ZIB 2017/zib-ComfortScale.xml]: MISSING CODE

File: Profiles - ZIB 2017/zib-ComfortScale.xml [hunk 8]
```diff
@@ -0,0 +1,413 @@
...
+      <definition value="A comment on the circumstances during the measurement." />
+      <alias value="Toelichting" />
+      <mapping>
+        <identity value="hcim-comfortscale-v1.0-2017EN" />
+        <map value="NL-CM:12.12.12" />
+        <comment value="Comment" />
+      </mapping>
+    </element>
+    <element id="Observation.component">
+      <path value="Observation.component" />
```
Comment (ArdonToonstra) [inline Profiles - ZIB 2017/zib-ComfortScale.xml]: Add component cardinalities!

**Pull request merged**